### PR TITLE
[12.0] REF l10n_it_fatturapa_in for cases of partners (contacts) with wrong fiscal code (=TIN)

### PR DESCRIPTION
**Pull request by eLBati**
First search by VAT number, otherwise by fiscal code.
Correctly check commercial partner

### CODE REVIEW

**Comment by sergiocorato** [comment]
consiglio queste modifiche 8c8dad364e67ff26fcf388f45badd9d6b91489db

**Comment by eLBati** [comment]
@sergiocorato @monen17 grazie, fatto.

File: l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py [hunk 1]
```diff
@@ -116,26 +116,32 @@ def getPartnerBase(self, DatiAnagrafici):
                 DatiAnagrafici.IdFiscaleIVA.IdPaese,
                 DatiAnagrafici.IdFiscaleIVA.IdCodice
             )
-        partners = partner_model.search([
-            '|',
-            ('vat', '=', vat or 0),
-            ('fiscalcode', '=', cf or 0),
-        ])
-        commercial_partner = False
+        partners = partner_model
```
Comment (GSLabIt) [inline l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py]: Can I ask why this? would't be better partners= False? Am I missing something?

Comment (eLBati) [reply l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py]: `partner_model` is an empty set. Using `partners= False` , line `len(partners)` would raise exception

Comment (GSLabIt) [reply l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py]: Agreed, in that case would be possible to check `if partners and len(partners) > 1`. Btw it's ok, thx

**Pull request merged**